apps/davi/src/clients/apollo.ts

```diff
@@ -13,36 +14,87 @@ export const subgraphClientsUris = {
   // testnets
   [SUPPORTED_DAVI_NETWORKS.ARBITRUM_TESTNET_ID]: '',
   [SUPPORTED_DAVI_NETWORKS.GOERLI_ID]: '',
-  [SUPPORTED_DAVI_NETWORKS.LOCALHOST_ID]:
-    'http://127.0.0.1:8000/subgraphs/name/dxdao/guilds',
+  [SUPPORTED_DAVI_NETWORKS.LOCALHOST_ID]: {
+    [SUPPORTED_SUBGRAPHS.Guilds]:
+      'http://127.0.0.1:8000/subgraphs/name/dxdao/guilds',
+    [SUPPORTED_SUBGRAPHS.Governance1_5]:
+      'http://127.0.0.1:8000/subgraphs/name/dxdao/dxgov-1-5',
+  },
 };
 
-const setupApolloClient = (network: SUPPORTED_DAVI_NETWORKS) =>
+const setupApolloClient = (
+  network: SUPPORTED_DAVI_NETWORKS,
+  subgraph: SUPPORTED_SUBGRAPHS
+) =>

```

Comment:
Since we're generating both Guilds and 1.5 subgraphs, maybe we can refactor this to take the network and return both clients